### ISSUE TITLE
Is it wrong for the discretized_mix_logistic_loss?

### ISSUE THREAD

**Issue by jiqizaisikao** [r9y9/wavenet_vocoder#34]
Hi,in the func:discretized_mix_logistic_loss
 log_probs = torch.sum(log_probs, dim=-1, keepdim=True) + F.log_softmax(logit_probs, -1)
why do this torch.sum(log_probs, dim=-1, keepdim=True)
 p=(π1*p1+π2*p2....)/(π1+π2...)
if F.log_softmax do this:log(π1/(π1+π2...)) log(π2/(π1+π2...)) ...
maybe should do like this:
log_probs = log_probs+ F.log_softmax(logit_probs, -1)
if modify it then the p is much bigger like exp(-5),otherwise the p is so small like exp(-50) it is unreasonable

**Comment by jiqizaisikao** [comment]
In my opinion,F.log_softmax output the normalized coefficient π1 π2....it maybe should sum the p1 p2....and π1 π2 first

**Comment by jiqizaisikao** [comment]
It seems that some other body found the same problem.Ignore this

**Comment by r9y9** [comment]
Sorry for the stupid bug. It's fixed now. Thanks for pointing this out!
